src/main/java/core/basesyntax/strategy/PurchaseHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class PurchaseHandler implements OperationHandler {
+    private static final String PURCHASE = "p";
```

Comment:
The constant `PURCHASE` is not necessary here as it's only used to compare with the operation type, which should already be known in the context of the `PurchaseHandler`. You can remove it to simplify the code.